hellocardboard-android/src/main/jni/hello_cardboard_app.cc

```diff
@@ -365,7 +365,8 @@ Matrix4x4 HelloCardboardApp::GetPose() {
   std::array<float, 3> out_position;
   long monotonic_time_nano = GetMonotonicTimeNano();
   monotonic_time_nano += kPredictionTimeWithoutVsyncNanos;
-  CardboardHeadTracker_getPose(head_tracker_, monotonic_time_nano,
+  CardboardHeadTracker_getPose(head_tracker_,monotonic_time_nano,
+                               kCardboardViewportOrientationLandscapeLeft,
```

Comment:
please also apply this change to hellocardboard-ios app, otherwise it will break.